### PR TITLE
Add profile for Martin

### PR DESCRIPTION
**Pull request by martinjohndyer**
Looks okay locally, hopefully all the formatting is correct.

### CODE REVIEW

**Review by Robadob — APPROVED** [review]
Seems good.

Reminds me I should update mine.

File: _people/martin-dyer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+othernames: Martin
+surname: Dyer
+role: Research Software Engineer
+
+---
+
+Martin joined the RSE team in 2024, and is based in the [School of Mathematical and Physical Sciences](https://www.sheffield.ac.uk/mps). He completed his PhD in astronomy at the University of Sheffield in 2020, after which he continued working as a postdoc in the [astronomy group](https://www.sheffield.ac.uk/mps/research/physics/astrophysics) before transitioning to a role as a research software engineer.
+
+Martin's primary focus is the Gravitional-wave Optical Transient Observer ([GOTO](https://goto-observatory.org/)) project, an international network of robotic survey telescopes which monitor the night sky for transient sources. During his PhD Martin developed the control and scheduling system for the GOTO telescopes, which are located on La Palma in the Canary Islands and in New South Wales, Australia. He is also a member of the [ULTRACAM](http://www.vikdhillon.staff.shef.ac.uk/ultracam/index.html)/[HiPERCAM](http://www.vikdhillon.staff.shef.ac.uk/hipercam/) collaboration, which operates several high-speed astronomical cameras located at some of the world's premier observatories.
```
Comment (ns-rse) [inline _people/martin-dyer.md:14]: Is it worth linking to the GOTO telescopes too?

**Review by ns-rse — APPROVED** [review]
Reads well, one minor thing I'd be inclined to do to fit with #701 should it ever be merged is to hard wrap long lines at 120 characters the length [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) is configured to use in that PR.

But that is very minor and I'm not sure #701 will ever see the light of day at this rate! :laughing:

**Comment by Robadob** [comment]
> But that is very minor and I'm not sure #701 will ever see the light of day at this rate! 😆

Don't lose hope, it took about 2 years for https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/446 (https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/154) to get merged .

File: _people/martin-dyer.md (hunk 1 above)
Comment (martinjohndyer) [reply _people/martin-dyer.md:14]: I did link to the main project page (https://goto-observatory.org/), the individual sites do have status pages (https://lapalma-observatory.warwick.ac.uk, https://goto-south.warwick.ac.uk) but they're not really for outside viewers and everything else should be on the .org page.

**Comment by martinjohndyer** [comment]
> Reads well, one minor thing I'd be inclined to do to fit with #701 should it ever be merged is to hard wrap long lines at 120 characters the length [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) is configured to use in that PR.
> 
> But that is very minor and I'm not sure #701 will ever see the light of day at this rate! 😆

Fair point, I did check some of the other files and there was a mix of wrapped and not. #701 doesn't seem to have changed any of the other files in `_people`, but I'm happy to edit it if that does go through.

**Pull request merged**